### PR TITLE
BCF-2167: mv status and profile from `node` to `admin`

### PR DESCRIPTION
**Pull request by krehermann**
In order to consistently scope TOML configuration

- hides but does not rm the admin subcmds and adds a warning that points to the updated location
- add subcmds to `admin` and reorganizes func locations to be in the same file

### CODE REVIEW

File: core/cmd/local_client.go [hunk 1]
```diff
@@ -114,6 +114,11 @@ func initLocalSubCmds(client *Client, devMode bool) []cli.Command {
 			Usage:  "Displays the health of various services running inside the node.",
 			Action: client.Status,
 			Flags:  []cli.Flag{},
+			Hidden: true,
+			Before: func(ctx *clipkg.Context) error {
+				client.Logger.Warnf("Command depreciated. Use `admin status` instead.")
```
Comment (jmank88) [inline core/cmd/local_client.go:119]:
```suggestion
				client.Logger.Warnf("Command deprecated. Use `admin status` instead.")
```

File: core/cmd/local_client.go [hunk 2]
```diff
@@ -131,6 +136,11 @@ func initLocalSubCmds(client *Client, devMode bool) []cli.Command {
 					Value: "/tmp/",
 				},
 			},
+			Hidden: true,
+			Before: func(ctx *clipkg.Context) error {
+				client.Logger.Warnf("Command depreciated. Use `admin profile` instead.")
```
Comment (jmank88) [inline core/cmd/local_client.go:141]:
```suggestion
				client.Logger.Warnf("Command deprecated. Use `admin profile` instead.")
```

**Review by jmank88 — COMMENTED** [review]
Changelog?

**Review by krehermann — COMMENTED** [review]
@jmank88 changelog added now. thanks

**Pull request merged**